.github/workflows/release_to_staging.yaml

```diff
@@ -0,0 +1,36 @@
+name: Release to staging
+on:
+  push:
+    branches: [ main ]
+jobs:
+  release-docker:
+    runs-on: ubuntu-latest
+    env:
+      REPO_NAME: datree
+      IMAGE_NAME: webhook-staging
+      DOCKER_IMAGE: datree/webhook-staging
+    name: Release container image to public registries
+    steps:
+      - uses: actions/checkout@v3
+      - name: Define release version
+        id: define_version
+        run: |-
+          OUTPUT_VERSION=$(bash scripts/get_last_short_commit_hash.sh)
+          echo "detected version = $OUTPUT_VERSION"
+          echo ::set-output name=version::$(echo $OUTPUT_VERSION)
+      - uses: docker/setup-buildx-action@v1
+      - name: Log in to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Build and push datree image to registries
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
```

Comment:
The build used to run using `scripts/build-docker-image.sh` which added some configurations, don't we want that?

Reply:
Add build args to prod release as it was in dockerhub build flow